src/main.js

```diff
@@ -35,53 +30,77 @@ process.on('uncaughtException', (exception: Error) => {
  */
 const container = new Container();
 defaultBindings(container, settings);
-
 const deviceServer = container.constitute('DeviceServer');
-deviceServer.start();
-
 const app = createApp(container, settings);
 
-const onServerStartListen = (): void =>
-  console.log(`express server started on port ${NODE_PORT}`);
+const startServers = () => {
+  deviceServer.start();
```

Comment:
This doesn't seem right but it probably works. I would have assumed that you would create a device server for each worker and moved the constitute code in here.